x-pack/legacy/plugins/uptime/public/components/functional/ping_list/expanded_row.tsx

```diff
@@ -52,20 +61,29 @@ export const PingListExpandedRowComponent = ({ ping }: Props) => {
   }
 
   // Show the body, if present
-  if (ping.http && ping.http.response && ping.http.response.body) {
+  if (ping.http?.response?.body) {
     const body = ping.http.response.body;
 
     listItems.push({
       title: i18n.translate('xpack.uptime.pingList.expandedRow.response_body', {
         defaultMessage: 'Response Body',
       }),
       description: (
-        <Fragment>
+        <>
           <BodyDescription body={body} />
-          <BodyExcerpt content={body.content || ''} />
-        </Fragment>
+          <EuiSpacer size={'s'} />
+          {body.content ? <BodyExcerpt content={body.content || ''} /> : <DocLinkForBody />}
```

Comment:
AFAICT we don't have a test case for this. The snapshot was updated, but there's no case where I see this rendered.

Reply:
I have added the case for this in test file.